### PR TITLE
add old files of former code for review

### PR DESCRIPTION
**Pull request by mckent05**
In this PR I have included the following :
- A HTML file of a former project for review
- A CSS file of a former project for review.
- A readme of a former project for review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    
+    <header id="header">
+      
+        <img width="300px" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="picture" id="header-img">
+        
+        <nav id="nav-bar">
+          
+            <a class="nav-link" href="#features">Features</a></li>
+            <a class="nav-link" href="#howitworks">How it Works</a></li>
+            <a class="nav-link"  href="#pricing">Pricing</a></li>
```
Comment (mckent05) [inline index.html:21]: Please wrap your links in a ul tag. wrapping them in the tag will help you position or style them better

File: index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      <h4>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase</h4> 
+    </div>
+    <div class="prem-mat">
+      <h1>Fast Shipping</h1>
+      <h4>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</h4> 
+    </div>
+    <div class="prem-mat">
+      <h1>Quality Assurance</h1>
+      <h4>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</h4> 
+    </div>
```
Comment (mckent05) [inline index.html:50]: Avoid using so many h1 tags in your HTML document. visit this site https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md for HTML and CSS best practices.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+      <div class="price-list">
+        <div>
+          <h1>TENOR TROMBONE</h1>
+        </div>
+        <p>$600</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <p>Lorem ipsum</p>
```
Comment (mckent05) [inline index.html:70]: Please kindly wrap in a ul tag. It helps makes your code cleaner

File: index.html [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+      </div>
+      <div class="price-list">
+        <div>
+          <h1>BASS TROMBONE</h1>
+        </div>
+        <p>$900</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum</p>
+        <p>Lorem ipsum dolor</p>
+        <p>Lorem ipsum</p>
```
Comment (mckent05) [inline index.html:83]: Kindly wrap in a ul tag for cleaner code.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+      </div>
+      <div class="price-list">
+        <div>
+          <h1>VALVE TROMBONE</h1>
+        </div>
+        <p>$1200</p>
+        <p>Plays similar to a trumpet</p>
+        <p>Great for Jazz band</p>
+        <p>Lorem ipsum dolor</p>
+        <p>Lorem ipsum</p>
```
Comment (mckent05) [inline index.html:96]: Kindly wrap in a ul tag for cleaner code.

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+    
+    
+    
+    
+    
+    
+}
+
+
+
```
Comment (mckent05) [inline css/style.css:20]: Please kindly remove all white spaces in your CSS files. You have too many of them.

**Review by mckent05 — COMMENTED** [review]
**Changes Required** ♻️

Good job so far on completing the project, but there are some changes required for your project to be complete.

Please follow my comment and effect the recommended changes.

**Comment by Davidosky007** [comment]
## Hi @mckent05 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! �
good-job-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**